src/rust/ensogl/lib/components/src/selector/model.rs

```diff
@@ -64,6 +64,14 @@ pub struct Model {
     /// Shape root that all other elements are parented to. Should be used to place the shapes as
     /// a group.
     pub root               : display::object::Instance,
+
+    background_color                  : Rc<RefCell<color::Rgba>>,
+    track_color                       : Rc<RefCell<color::Rgba>>,
+    background_left_corner_roundness  : Rc<Cell<bool>>,
+    background_right_corner_roundness : Rc<Cell<bool>>,
+    padding                           : Rc<Cell<f32>>,
+
+    pub app : Application,
```

Comment:
Usually such parameters are just the FRP outputs of components. For example: the ListView does not keep its size and scroll position in the model, but as the frp outputs anyone could read.

Reply:
So, you would change it to be more consistent with those components?

Reply:
Yes, if there are no specific reason for keeping them as they are. But if there is, please describe that reason in a comment.

Reply:
It is not possible the way how things are currently structured. There is not FRP network available. I would have to refactor things. I don't think it's worth it.